modules/serverless-rn-1-29-0.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-29-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.29
+
+{ServerlessProductName} 1.29 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-29-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.8.
+* {ServerlessProductName} now uses Knative Eventing 1.8.
+* {ServerlessProductName} now uses Kourier 1.8.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.8.
+* {ServerlessProductName} now uses Knative Kafka 1.8.
+* The `kn func` CLI plug-in now uses `func` FIXME.
+
+* Beginning with {ServerlessProductName} 1.29, the different product versions are available as follows:
+** The latest release is available through the `latest` channel.
```

Comment:
The latest release is actually available through the `stable` channel. The other named channel is currently `stable-1.29`. See this line: https://github.com/openshift-knative/serverless-operator/blob/release-1.29/olm-catalog/serverless-operator/Dockerfile#L11
We have only "stable" and "stable-1.29".